x-pack/plugins/apm/public/components/app/ServiceDetails/AlertIntegrations/index.tsx

```diff
@@ -96,17 +109,22 @@ export function AlertIntegrations(props: Props) {
           : []),
       ],
     },
+
+    // transaction duration panel
```

Comment:
I can remove these comments again. Adding them for my own sake but I can see how they are somewhat noisy/redundant.